runtime/cmake/omr_config.cmake

```diff
@@ -25,3 +25,6 @@ set(OMR_DDR OFF CACHE INTERNAL "")
 set(OMR_EXAMPLE OFF CACHE INTERNAL "")
 set(OMR_FVTEST OFF CACHE INTERNAL "")
 set(OMR_GC ON CACHE INTERNAL "")
+
+# Force OMR options to match mirrored j9vm options
+set(OMR_OPT_CUDA ${J9VM_OPT_CUDA} CACHE BOOL "" FORCE)
```

Comment:
I think we should remove `J9VM_OPT_CUDA` and reference `OMR_OPT_CUDA` instead; `OMR_OPT_CUDA` should be enabled on the cmake command-line when `configure --enable-cuda` is used.